lib/controllers/getTarball.js

```diff
@@ -4,27 +4,43 @@ function getTarball(respond, route, unpm) {
   var version = route.params.version
   var name = route.params.name
   var started = false
+  var getTarballAsync = unpm.Package.getTarballAsync
+  var getTarballSync = unpm.Package.getTarball
```

Comment:
I would rename these to `getTarballStreamSync` `getTarballStreamAsync` since it is not getting the actual tarball synchronously, just a stream that will eventually produce the tarball.